README.md

```diff
@@ -1 +1,72 @@
-# To-do-list-review
\ No newline at end of file
+
+# TO-DO-LIST
+
+> This project has as aim to teach us about webpack and generating a list of tasks, add, delete, drag and drop them in their box.
+
+
+![todo](https://user-images.githubusercontent.com/72297212/159683357-0d7ac584-78a0-473d-8a7d-e74e12dc1240.PNG)
+
+
+
+## Built With
+
+- HTML, CSS, JAVASCRIPT & WEBPACK
+
+## Live
+
+[Live Demo](https://ciraganenicole.github.io/To-do-list/dist/index.html)
+
+## Getting Started
```

Comment:
Kindly add more details on how to set up a copy of your project for example "what to do after cloning it or even the steps  that lead to the git clone"
eg cd to the folder in which you would like to clone the repository
- Clone the repository by running git clone"repo-URL" within that folder